pages/blog/february-2021-at-asyncapi.md

```diff
@@ -0,0 +1,188 @@
+- [How **generator-model-sdk** simplify JSON Schema](https://github.com/asyncapi/generator-model-sdk/blob/master/docs/simplification.md)
+- [How model generator works and how to write a new one](https://github.com/asyncapi/generator-model-sdk/blob/master/docs/generators.md)
+- [How presets (customization) works and how to write a custom preset](https://github.com/asyncapi/generator-model-sdk/blob/master/docs/customization.md)
+
+## Community continues to grow
+
+In February we reached 1600 followers on Twitter.
+
+<TwitterTweetEmbed
+  tweetId='1362016089207635968'
+  options={{
+    cards: 'hidden',
+    width: 500,
+    align: 'center'
+  }}
+/>
+
+A great thing to see is that the community from [LinkedIn](https://www.linkedin.com/feed/update/urn:li:activity:6769647615823945728/) grew to 900, and our news shared there also get more visibility.
+
+We are very noisy :sweat_smile:
+
+## Other features
+
+- Thanks to the support of [Daniel CHU](https://www.linkedin.com/in/daniel-chu-dc/) that joined us during last Hacktoberfest now the JavaScript parser also validates the examples of server variables. For more details, check [1.4](https://github.com/asyncapi/parser-js/releases/tag/v1.4.0) release.
+- Once again, thanks to [Ludovic Dussart](https://twitter.com/ldussart) from Ineat Lab, we have additional features, these time in HTML and Markdown template. Thanks to the new **version** parameter, you can overwrite the version of the application specified in the AsyncAPI file under **info.version**. Useful in CI/CD when your service version is not maintained in the #AsyncAPI file but pom.xml or package.json
+- Thanks to [Mike Ralphson](https://twitter.com/PermittedSoc), we have a new **frontMatter** parameter in the Markdown template. It lets you specify an external file with a custom front-matter that should be included in generated Markdown during generation. Very useful for static site generators' users.
+
+  ```bash
+  # 1. Get AsyncAPI Generator
+  npm install -g @asyncapi/generator
+
+  # 2. Create a file with frontmatter
+  cat > ssg.yml << EOF
+  title: AsyncAPI Documentation
+  layout: asyncapi
+  permalink: /asyncapi-docs
+  EOF
+
+  # 3, Generate Markdown file that includes the frontmatter
+  ag https://bit.ly/asyncapi @asyncapi/markdown-template -o output -p frontMatter=ssg.yml
+
+  # 4. Check out the output
+  cat output/asyncapi.md
+  ```
+- [Maciej Urbanczyk](https://www.linkedin.com/in/maciej-urba%C5%84czyk-909547164/) and [I](https://twitter.com/derberq) (I mean me :smiley:) pushed some features to the AsyncAPI Generator:
+  - It supports the latest Node.js 15 and npm 7
+  - You can now install generator templates globally. For more details, read [this](https://github.com/asyncapi/generator/#global-templates-installed-with-yarn-or-npm) new section in the readme.
+  - It is now much easier to generate multiple files using the new React render engine
```

Comment:
Please add reference to docs -> https://github.com/asyncapi/generator/blob/master/docs/authoring.md#react and for example in real template: https://github.com/asyncapi/template-for-generator-templates/blob/master/template/schemas/schema.js#L10